runtime/opt/taupage/init.d/02-register-td-agent.py

```diff
@@ -147,21 +147,26 @@ def update_configuration_from_template():
 
 
 if __name__ == '__main__':
+    hostname = boto.utils.get_instance_metadata()['local-hostname'].split('.')[0]
     config = get_config()
     logging_config = config.get('logging')
+    if logging_config:
+        if not logging_config.get('fluentd_enabled'):
+            logger.info('Fluentd disabled; skipping Fluentd initialization')
+            raise SystemExit()
     if not logging_config:
-        logger.info('Found no logging section in senza.yaml')
-        raise SystemExit()
-    else:
+        logger.info('Found no logging section in senza.yaml; enable dafault logging to s3')
         try:
-            with open('/etc/cron.d/get_fluentd_metrics', 'w') as file:
-                file.write('#!/bin/bash\n')
-                file.write('PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin\n')
-                file.write('* * * * * root /opt/taupage/bin/get-fluentd-metrics.sh\n')
+            with open('/var/local/textfile_collector/fluentd_default_s3.prom', 'w') as file:
+                file.write('fluentd_default_s3_logging{tag=\"td-agent\",hostname=\"'+hostname+'\"} 1.0\n')
```

Comment:
String interpolation is cleaner with the `string.format()` function

Reply:
Done